core/admin/mailu/ui/forms.py

```diff
@@ -41,6 +41,15 @@ def __call__(self, form, field):
         if not pattern.match(field.data.replace(" ", "")):
             raise validators.ValidationError(self.message)
 
+class MultipleFoldersVerify(object):
```

Comment:
Please add a docstring that explains what this class checks. In other words explain what the regular expression verifies.

Reply:
Fixed in c79e8d3852aac0c57f88bb2a930ffe68e761403e ... this has been introduced in 6474108